rwslib/extras/audit_event/main.py

```diff
@@ -1,10 +1,14 @@
 # -*- coding: utf-8 -*-
 __author__ = 'isparks'
 
-from rwslib import RWSConnection
 from rwslib.rws_requests.odm_adapter import AuditRecordsRequest
-from urlparse import urlparse, parse_qs
-from parser import parse
+
+try:
```

Comment:
damn, info was based on: http://python3porting.com/stdlib.html#urllib-urllib2-and-urlparse

changing :)